### ISSUE TITLE
[Feature Request] Sorting of Keys

### ISSUE THREAD

**Issue by ZimmerD** [CSBiology/DeedleExtensions#5]
Sorting of row and column keys can be really cumbersome, I think it would be worthwhile to extend the existing functionality.

**Comment by HLWeil** [comment]
I totally agree. For me I think especially a `sortRowsByKeysBy` and a `sortColsByKeysBy` function are missing. What else would you suggest?
